packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -334,9 +343,11 @@ class RetainingPathWidget extends StatelessWidget {
         for (RetainingObject object in retainingPath.elements!.sublist(1))
           Row(
             children: [
-              SelectableText(
-                _retainingObjectDescription(object),
-                style: Theme.of(context).fixedFontStyle,
+              Flexible(
```

Comment:
why do all these rows only have a single child? makes me wonder why we need the rows at all

Reply:
I think it is so the contents can be flexible, when the text is too long